app/web/components/Alert.tsx

```diff
@@ -6,7 +6,7 @@ import classNames from "classnames";
 import React from "react";
 import makeStyles from "utils/makeStyles";
 
-import { grpcErrorStrings, ObscureGrpcErrorMessages } from "../constants";
+import { grpcErrorStrings, ObscureGrpcErrorMessages } from "../appConstants";
```

Comment:
```suggestion
import { grpcErrorStrings, ObscureGrpcErrorMessages } from "appConstants";
```
Wouldn't this also work now? I remember importing `constants` was giving me funny results in our current app with Create React App, as it seems to clash with a module named the same in Node?

Reply:
Yes